cli/internal/globby/globby.go

```diff
@@ -24,8 +38,10 @@ func GlobFiles(basePath string, includePatterns []string, excludePatterns []stri
 
 	includePattern := "{" + strings.Join(include, ",") + "}"
 	excludePattern := "{" + strings.Join(exclude, ",") + "}"
-	_ = fs.Walk(basePath, func(p string, isDir bool) error {
-		if val, _ := doublestar.PathMatch(excludePattern, p); val {
+
+	_ = afs.Walk(basePath, func(path string, info os.FileInfo, err error) error {
+		var isDir = info.IsDir()
+		if val, _ := doublestar.PathMatch(excludePattern, path); val {
```

Comment:
ditto re: error

Reply:
I'm going to defer this since it is the present state and the immediate followup will remove this code.